src/Xamarin.Android.Build.Utilities/Sdks/MonoDroidSdkBase.cs

```diff
@@ -8,64 +8,31 @@ namespace Xamarin.Android.Build.Utilities
 {
 	abstract class MonoDroidSdkBase
 	{
-		protected readonly static string DebugRuntime = "Mono.Android.DebugRuntime-debug.apk";
 		protected readonly static string ClassParseExe = "class-parse.exe";
-		protected readonly static string GeneratorScript = "generator";
-
-		// I can never remember the difference between SdkPath and anything else...
-		[Obsolete ("Do not use.")]
-		public string SdkPath { get; private set; }
```

Comment:
> Doing this we break API compatibility with android-tools. 

Do we need to maintain API compatibility? That's a separate repo, and will continue to exist.

> Should these changes been pushed to the xamarin-android-tools repo?

Probably. :-/

> AFAIK the end goal is to move all the SDK resolving code to xamarin-android-tools and have `Xamarin.Android.Build.Utilities` and android-tools use that so we have one common code base.

That is and continues to be an end goal.

What we don't yet know is what that end goal even looks like.

For example, in the New World Order this PR moves us toward, there is no longer a reason for most of `MonoDroidSdk`. What's there is logic to find the XA profile assemblies based on "runtimePath", which is *completely redundant*, because that's what the `bclPath` parameter to `MonoDroidSdk.Refresh()` provides.

The only reason it isn't any smaller is because we still need the logic to know which Android API levels we support -- `MonoDroidSdk.GetFrameworkVersionForApiLevel()` and related -- and *that* is better handled through PR #599 (once it actually works).

Meaning the end goal could be the complete removal of `MonoDroidSdk` and related types, because we don't need them anymore.

Is that not a wonderful state of affairs?